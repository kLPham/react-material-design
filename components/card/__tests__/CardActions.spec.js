import React from 'react';
import { shallow } from 'enzyme';
import CardActions from '../cardActions';

describe('<CardActions />', () => {
    it('should render', () => {
        const cardActions = shallow(
            <CardActions><button>Click</button></CardActions>,
    );
        expect(cardActions).toMatchSnapshot();
    });
    it('should render vertical', () => {
        const vertical = true;
        const cardActions = shallow(
            <CardActions vertical={vertical}><button>Click</button></CardActions>,
    );
        expect(cardActions).toMatchSnapshot();
    });
});
