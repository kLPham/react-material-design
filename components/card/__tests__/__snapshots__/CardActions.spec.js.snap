// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardActions /> should render 1`] = `
<section
  className="mdc-card__actions"
>
  <button>
    Click
  </button>
</section>
`;

exports[`<CardActions /> should render vertical 1`] = `
<section
  className="mdc-card__actions mdc-card__actions--vertical"
>
  <button>
    Click
  </button>
</section>
`;
